ote/src/clj/ote/services/taxi_index.clj

```diff
@@ -0,0 +1,125 @@
+    [ring.middleware.anti-forgery :as anti-forgery]
+    [ote.transit :as transit]
+    [ote.services.transport-operator :as transport-operator]))
+
+(def stylesheets [; Normalize CSS across browsers to be visually similar, works as base for all other styles
+                  {:href "css/taxiui/normalize.css"}
+                  ; Taxi UI global base definitions
+                  {:href "css/taxiui/styles.css"}
+                  ; Google fonts preconnects, improves performance
+                  {:rel "preconnect" :href "https://fonts.googleapis.com"}
+                  {:rel "preconnect" :href "https://fonts.gstatic.com" :crossorigin "true"}
+                  ; Fintraffic primary font
+                  {:href "https://fonts.googleapis.com/css?family=Public+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900" :rel "stylesheet"}
+                  ; Fintraffic secondary font
+                  {:href "https://fonts.googleapis.com/css2?family=Piazzolla:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" :rel "stylesheet"}])
+
+(defn ote-js-location [dev-mode?]
+  (str "js/ote"
+       (when-not dev-mode?
+         (str "-" (:current-revision-sha (current-revision-sha))))
+       ".js"))
+
+(def favicons
+  [{:rel "apple-touch-icon" :sizes "180x180" :href "/favicon/apple-touch-icon.png?v=E6jNQXq6yK"}
+   {:rel "icon" :type "image/png" :sizes "32x32" :href "/favicon/favicon-32x32.png?v=E6jNQXq6yK"}
+   {:rel "icon" :type "image/png" :sizes "16x16" :href "/favicon/favicon-16x16.png?v=E6jNQXq6yK"}
+   {:rel "manifest" :href "/favicon/manifest.json?v=E6jNQXq6yK"}
+   {:rel "mask-icon" :href "/favicon/safari-pinned-tab.svg?v=E6jNQXq6yK" :color "#5bbad5"}
+   {:rel "shortcut icon" :href "/favicon/favicon.ico?v=E6jNQXq6yK"}])
+
+(def dev-favicons
+  [{:rel "icon" :type "image/png" :sizes "32x32" :href "/favicon/favicon-dev-32x32.png"}
+   {:rel "icon" :type "image/png" :sizes "16x16" :href "/favicon/favicon-dev-16x16.png"}])
+
+(defn translations [language]
+  [:script#ote-translations {:type "x-ote-translations"}
+   (transit/clj->transit
+     {:language language
+      :translations (localization/translations language)})])
+
+(defn user-info [db user]
+  [:script#ote-user-info {:type "x-ote-user-info"}
+   (transit/clj->transit
+     (when user
+       (transport-operator/get-user-transport-operators-with-services db (:groups user) (:user user))))])
+
+(defn index-page [db user config]
+  (let [dev-mode? (:dev-mode? config)
+        testing-env? (:testing-env? config)
+        flags (str/join "," (map name (:enabled-features config)))
+        matomo-config (:matomo config)]
+    [:html
+     [:head
+      (if testing-env?
+        (for [f dev-favicons]
+          [:link f])
+        (for [f favicons]
+          [:link f]))
+      [:meta {:name "theme-color" :content "#ffffff"}]
```

Comment:
Semantic color could used here instead of hardcoded `"#ffffff"`, though it's hardcoded in `ote.services.index` as well.